xhr.bs

```diff
@@ -2031,6 +2041,12 @@ attributes initialized to false, so it is suggested that for consistency all
 {{ProgressEvent}} interface do the same.
 
 
+<h3 id=feature-policy>Feature Policy Integration</h3>
+
+<p>This specification defines a <a>policy-controlled feature</a> identified by
+the string <code><dfn>sync-xhr</dfn></code>. Its <a>default allowlist</a> is
+<code>*</code>.
```

Comment:
More can be on one line here. Please use 100 columns for new/modified text.

Reply:
Done.